mypyc/genops.py

```diff
@@ -109,6 +110,33 @@ def is_trait(cdef: ClassDef) -> bool:
                if isinstance(d, NameExpr))
 
 
+def update_parent_vtable(entries: VTableEntries, cls: ClassIR, is_trait: bool) -> VTableEntries:
+    updated = []
```

Comment:
Add docstring and/or rename the function -- it wasn't immediately obvious what the purpose of this function is.

Reply:
Cleaned it up to just take the parent/trait as an argument, too